src/lib/abacus/logger.ts

```diff
@@ -9,7 +9,10 @@ class AbacusLogger implements Omit<AbacusLoggingProtocol, '__doNotUseOrImplement
   }
 
   e(tag: string, message: string) {
-    if (import.meta.env.VITE_ENABLE_ABACUS_LOGGING) {
+    if (
+      import.meta.env.VITE_ENABLE_ABACUS_LOGGING ||
+      import.meta.env.VITE_ABACUS_LOG_LEVEL === 'error'
```

Comment:
abacus logging is really noisy, this is nice if we want to only see certain logs. this isn't really necessary for this PR so i can break it out but i figured why not

Reply:
nit what if we had `import.meta.env.VITE_ENABLE_ABACUS_DEBUGGER_LOGGING` (on line 5) and then replaced this line with `import.meta.env.VITE_ENABLE_ABACUS_ERROR_LOGGING`?

Reply:
yeah ideally we'll get to something like that. for now i figured i didn't want to break other peoples' env set ups without some sort of announcement. will definitely get to it at some point.


FWIW - i'm not a huge fan of having boolean env vars unless we abstract env vars through a `settings` file

env vars are always imported as strings, so you could set `VITE_ENABLE_ABACUS_LOGGING=false` and it would still evaluate to a truthy value 👀 

ideally we write a `settings.ts` file that converts any boolean env var values.

also, for this specific var [i decided to follow logging standards](https://sematext.com/blog/logging-levels/).
- so ERROR is more specific than WARN, followed by DEBUG, followed by INFO.

i'm definitely open to changing this though! just thought until we discussed and came to consensus it just made sense to follow existing log level patterns